### PR TITLE
BaSP-MR-54: Footer component

### PR DESCRIPTION
**Pull request by MatiasSauvanet**
Based in figma desing.

### CODE REVIEW

File: src/Components/Footer/index.jsx [hunk 1]
```diff
@@ -1,61 +1,35 @@
...
+              className={styles.socialIcon}
+              src={'./assets/images/FacebookLogo.svg'}
+            />
+          </a>
+        </div>
+        <div className={styles.copyright}>
+          Copyright © {new Date().getFullYear()} MegaRocket S.A. All rights reserved.
         </div>
       </div>
     </footer>
```
Comment (Dannylez) [inline src/Components/Footer/index.jsx:35]: Be careful, the file we are going to use is the "footer" located in the "shared" folder.

File: public/assets/images/FacebookLogo.svg
Comment (DelfiLT) [inline public/assets/images/FacebookLogo.svg:1]: Use kebab case with images files name

File: public/assets/images/InstagramIcon.svg
Comment (DelfiLT) [inline public/assets/images/InstagramIcon.svg:1]: And here

File: public/assets/images/TwitterIcon.svg
Comment (DelfiLT) [inline public/assets/images/TwitterIcon.svg:1]: And here

**Review by ukiweskamp — COMMENTED** [review]
You have to use the footer that is inside the shared folder

**Review by DelfiLT — APPROVED** [review]

**Review by ukiweskamp — APPROVED** [review]

**Pull request merged**